lib/audited/auditor.rb

```diff
@@ -261,6 +271,18 @@ def enable_auditing
       def audit_as( user, &block )
         Audited.audit_class.as_user( user, &block )
       end
+
+      def auditing_enabled
+        if (val = Thread.current[const_get('AUDIT_VAR_NAME')]).nil?
```

Comment:
Going through the `audited` code I noticed that `Audited.store` exists for dealing with thread safe state, so maybe this check could be written as `Audited.store.fetch("#{name.tableize}_auditing_enabled", true)` or something similar to let the `Audited.store` method know about thread variables instead of having it here.